src/test/java/characterpicker/infrastructure/CharacterParserTest.java

```diff
@@ -19,7 +19,7 @@ void shouldThrowExceptionWhenSpacesSeparatedByCommaIsProvided() {
     @Test
     void shouldParseCharacter() {
         // GIVEN
-        String singularCharacter = "Isaac,1";
+        String singularCharacter = "Isaac,1; ";
```

Comment:
Patrząc po tym jak działa kod, to spodziewałbym się, że ten test się wywali bo masz 

```java
        if (unparsedCharacter.endsWith(";")){
            throw new IllegalArgumentException("Completion required. If you don't have one type 'none'");
        }
```

Co w tym przypadku nie jest spełnione, bo masz na końcu spacje, a nie none.